### PR TITLE
Add vapor pressure for compounds batch3

### PR DESCRIPTION
**Pull request by sumitnegi7**
Sir @maneetgoyal  can you verify the key names for number and string ones are according to your suggestion. Rest i will recheck the data once.

### CODE REVIEW

File: src/data/vapor-pressure-constants.ts [hunk 1]
```diff
@@ -18,6 +18,308 @@ export const DIPPRVaporPressureDictionary: Record<
...
+    C3: -3.4258,
+    C4: 5.63e-17,
+    C5: 6,
+    minimumTemperature: 175.45,
+    maximumTemperature: 416.25,
+    pressureAtMinimumTemperature: 8.84e2,
+    pressureAtMaximumTemperature: 6.759e6,
+  },
+
+  Chloropropane1: {
```
Comment (maneetgoyal) [inline src/data/vapor-pressure-constants.ts:264]:
```suggestion
  `1Chloropropane`: {
```

Rename past entries also using this format.

Comment (sumitnegi7) [reply src/data/vapor-pressure-constants.ts:264]: Okay sir

File: src/data/vapor-pressure-constants.ts [hunk 2]
```diff
@@ -603,6 +603,306 @@ export const DIPPRVaporPressureDictionary: Record<
...
+    formula: 'C4H7N',
+    CAS: '109-74-0',
+    C1: 60.6576,
+    C2: -6404.32,
+    C3: -5.49286,
+    C4: 1.13e-17,
+    C5: 6,
+    minimumTemperature: 161.3,
+    maximumTemperature: 585.4,
+    pressureAtMinimumTemperature: 9.41e4,
```
Comment (maneetgoyal) [inline src/data/vapor-pressure-constants.ts:693]:
```suggestion
    pressureAtMinimumTemperature: 9.41e-4,
```

File: src/data/vapor-pressure-constants.ts [hunk 3]
```diff
@@ -603,6 +603,306 @@ export const DIPPRVaporPressureDictionary: Record<
...
+    C3: -8.5766,
+    C4: 6.8465e-6,
+    C5: 2,
+    minimumTemperature: 250.33,
+    maximumTemperature: 556.35,
+    pressureAtMinimumTemperature: 1.12e3,
+    pressureAtMaximumTemperature: 4.544e6,
+  },
+
+  CarbonTetraFlouoride: {
```
Comment (maneetgoyal) [inline src/data/vapor-pressure-constants.ts:757]:
```suggestion
  CarbonTetraFluoride: {
```

File: src/data/vapor-pressure-constants.ts [hunk 4]
```diff
@@ -603,6 +603,306 @@ export const DIPPRVaporPressureDictionary: Record<
...
+    C3: -4.8814,
+    C4: 0.000075673,
+    C5: 2,
+    minimumTemperature: 68.15,
+    maximumTemperature: 132.92,
+    pressureAtMinimumTemperature: 1.54e4,
+    pressureAtMaximumTemperature: 3.494e6,
+  },
+
+  CarbonTetraChloride: {
```
Comment (maneetgoyal) [inline src/data/vapor-pressure-constants.ts:742]:
```suggestion
  CarbonTetrachloride: {
```

**Review by maneetgoyal — APPROVED** [review]

**Pull request merged**